Platform/Xiaomi/sm7150/surya.fdf.inc

```diff
@@ -0,0 +1,35 @@
+// per-device BSP DXEs
+
+FILE DRIVER = 8e9bd160-b184-11df-94e2-0800200c9a66 {
+    SECTION DXE_DEPEX = Platform/EFI_Binaries/Drivers/sm7150/DALSys/DALSys.depex
+    SECTION PE32 = Platform/EFI_Binaries/Drivers/Devices/surya/DALSys/DALSys.efi
+    SECTION UI = "DALSys"
+}
+
+FILE DRIVER = 5bd181db-0487-4f1a-ae73-820e165611b3 {
+    SECTION DXE_DEPEX = Platform/EFI_Binaries/Drivers/sm7150/ButtonsDxe/ButtonsDxe.depex
+   SECTION PE32 = Platform/EFI_Binaries/Drivers/Devices/surya/ButtonsDxe/ButtonsDxe.efi

```

Comment:
Unneeded, ButtonsDxe is already patched.

Reply:
sorry jeje